assets/accents/slav.ron

```diff
@@ -0,0 +1,50 @@
+(
+    name: "slav",
+    description: "Always drunk",
+    rules: (
+        words: [
+            ("my", Simple("our")),
+            ("friend", Simple("comrade")),
+            ("(enemy|foe)", Simple("american pig")),
+            ("(fuck|shit)", Any([
+                Simple("blyat"),
+                Simple("cyka"),
+            ])),
+            ("usa", Simple("американские захватчики")),
+            ("we are being attacked", Simple("нас атакуют")),
```

Comment:
I'm not sure replacing text with a foreign language is good. Text should still be understandable after an accent is applied imo.

Reply:
slav accent isnt as good as i wanted it to be anyway, i removed it completely. itll stay in separate repo in case someone wants to improve it